no-computer.md

```diff
@@ -1,44 +1,52 @@
-# No computer project
-
+# Violin
 
 ## The user and a language
 This section describes who the project would serve and why a language might be a
 good way to meet their needs.
 
+Violin is a hard instrument. Rather than having a separated fingerbaord for each note, like frets on a guitar, violin simply has a continuous fingerboard. The violin produces different notes based off of where fingers are placed and in what combination. Having a language that emulates what happens when some combination of fingers are placed would allow users to focus more on the quality of the music rather than on producing the right sound. My computer-free project would be for novices learning violin for the first time, or people who don't have time to formally learn violin but have composition ideas. 
 
 ### What's the need?
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+This language would eliminate the need to practice (for years usually) to train both the ear to identify notes and the fingers to produce the correct notes. This language would help beginners understand proper finger placement as well as amateur composers that simply want to create a violin piece. 
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+This "DSL" is appropriate because all violin players use the same "DSL". Therefore, when players play the same piece, they are similar in sound. However, this "DSL" also has flexibility in that there are many ways to play the same piece. Though the notes may be exactly the same, the "DSL" allows the user to express their personal preference and tastes which is reflected in their sound.
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+Growing up, learning violin was extremely frustrating. The concept of having to train the ear to recognize pitches in order to properly play the correct notes was baffling to me: I often complained about how the piano had discrete notes while the violin did not. This idea is exciting because a) I now regret having quit violin after maturing and recognizing how fun of a hobby instruments can be and b) so no one has to go through the struggle I did.
 
 ### Domain
 _Describe the project's domain in five words._
 
+Violin is a hard instrument
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
 
+The user would give some placement of their fingers which would then produce some sound. Because the domain is extremely specific, this is pretty much the only way to interact with the problem domain, and therefore the right way.
 
```

Comment:
I have limited knowledge about the violin, but would you consider the bow at all when discussing the interface? I see that you discuss it in the next question, but does it serve any other function in this analogy other than in running the program?